doc/manual/team.rst

```diff
@@ -225,6 +225,9 @@ TOO-LATE
 
 The judges may have prepared multiple test files for each problem.
 DOMjudge will report back the first non-correct result as verdict.
+*DOMjudge will always output the verdict on the first ordered testcase.
```

Comment:
I'm not very happy about this wording, so feel free to fully rewrite to actually convey what I'm trying to say.

Reply:
This is also not true if you change the results prio

Reply:
Just to be clear you mean:

RTE: prio 99
WA: prio 98
CORRECT: prio 1

or during the contest changing the ordering of the testcases? The latter is to specific I think... the first I did not think about.

Reply:
The former